python/friesian/democode/train_wnd.py

```diff
@@ -0,0 +1,121 @@
+                          names=["user", "gender", "age", "occupation", "zipcode"])
+    user_tbl = FeatureTable.from_pandas(user_df).cast(["user"], "int")
+    user_tbl.cache()
+
+    item_df = pd.read_csv(data_dir + "/ml-1m/movies.dat", encoding="ISO-8859-1",
+                          delimiter="::", names=["item", "title", "genres"])
+    item_tbl = FeatureTable.from_pandas(item_df).cast("item", "int")
+    item_tbl.cache()
+
+    user_tbl = user_tbl.fillna('0', "zipcode")
+
+    user_stats = ratings_tbl.group_by("user", agg={"item": "count", "rate": "mean"}) \
+        .rename({"count(item)": "user_visits", "avg(rate)": "user_mean_rate"})
+    user_stats, user_min_max = user_stats.min_max_scale(["user_visits", "user_mean_rate"])
+
+    item_stats = ratings_tbl.group_by("item", agg={"user": "count", "rate": "mean"}) \
+        .rename({"count(user)": "item_visits", "avg(rate)": "item_mean_rate"})
+    item_stats, user_min_max = item_stats.min_max_scale(["item_visits", "item_mean_rate"])
+
+    user_tbl, inx_list = user_tbl.category_encode(["gender", "age", "zipcode", "occupation"])
+    item_tbl, item_list = item_tbl.category_encode(["genres"])
+
+    item_size = item_stats.select("item").distinct().size()
+    ratings_tbl = ratings_tbl.add_negative_samples(item_size=item_size, item_col="item",
+                                                   label_col="label", neg_num=1)
+
+    user_tbl = user_tbl.cross_columns([["gender", "age"], ["age", "zipcode"]], [50, 200])
+
+    user_tbl = user_tbl.join(user_stats, on="user")
+    item_tbl = item_tbl.join(item_stats, on="item")
+    full = ratings_tbl.join(user_tbl, on="user") \
+        .join(item_tbl, on="item")
+
+    train_tbl, test_tbl = full.random_split([0.8, 0.2], seed=1)
+    train_count, test_count = train_tbl.size(), test_tbl.size()
+
+    stats = full.get_stats(cat_cols, "max")
+    wide_dims = [stats[key] for key in wide_cols]
+    wide_cross_dims = [stats[key] for key in wide_cross_cols]
+    embed_dims = [stats[key] for key in embed_cols]
+    indicator_dims = [stats[key] for key in indicator_cols]
+    column_info = ColumnFeatureInfo(wide_base_cols=wide_cols,
+                                    wide_base_dims=wide_dims,
+                                    wide_cross_cols=wide_cross_cols,
+                                    wide_cross_dims=wide_cross_dims,
+                                    indicator_cols=indicator_cols,
+                                    indicator_dims=indicator_dims,
+                                    embed_cols=embed_cols,
+                                    embed_in_dims=embed_dims,
+                                    embed_out_dims=[8] * len(embed_dims),
+                                    continuous_cols=num_cols,
+                                    label="label")
+    conf = {"column_info": column_info, "hidden_units": [20, 10], "lr": 0.001}
+
+    est = Estimator.from_keras(model_creator=model_creator, config=conf)
```

Comment:
do not pass `conf` in `from_keras`

define a method `model_creator` that direct use `conf` to create a W&D model